Client.xcodeproj/project.pbxproj

```diff
@@ -5163,7 +5163,7 @@
 				EB9A179B20E69A7F00B12184 /* ThemeManager.swift in Sources */,
 				EBFDB790211C83A5005CCA2F /* BrowserViewController+FindInPage.swift in Sources */,
 				EBB89509219398E500EB91A0 /* TabContentBlocker+ContentScript.swift in Sources */,
-				0B3E7D951B27A7CE00E2E84D /* AboutHomeHandler.swift in Sources */,
+                               0B3E7D951B27A7CE00E2E84D /* AboutHomeHandler.swift in Sources */,
```

Comment:
Looks like there is an extra space although when editing the file it showed the same alignment...will modify it

Reply:
@isabelrios 
I'm not sure about this .. i thought these entitlements had no prefix at all, but if they already had `$(SRCROOT)` then that should be turned into an absolute path?

(I don't know if `$(PROJECT_ROOT)` is actually a thing - i was just using that as an example)